laser_scan_matcher/src/laser_scan_matcher.cpp

```diff
@@ -162,6 +166,12 @@ void LaserScanMatcher::initParams()
   if (!nh_private_.getParam ("use_vel", use_vel_))
     use_vel_ = false;
 
+  // **** Are velocity input messages stamped?
+  // if false, will subscribe to Twist msgs on /vel
+  // if true, will subscribe to TwistStamped msgs on /vel
+  if (!nh_private_.getParam ("stamped_vel", stamped_vel_))
```

Comment:
I added a parameter in demo.launch to raise awareness of how to activate the proposed feature. https://github.com/ccny-ros-pkg/scan_tools/pull/35/files#diff-da85c9e3d31b840300d30555c2129fc8R6